keps/sig-storage/3333-reconcile-default-storage-class/README.md

```diff
@@ -269,10 +275,13 @@ need to create/delete default SCs, which could affect other tests running in
 parallel that may expect that a default SC exists (typically StatefulSet e2e
 tests).
 
-We plan to add a few `[Disruptive]` `[Serial]` tests with the new behavior as
-"smoke" tests of the new behavior, still, most of the tests will be integration
+We added a `[Disruptive]` `[Serial]` test with the new behavior as
+"smoke" test of the new behavior, still, most of the tests will be integration
 ones.
 
+Test:
+https://github.com/kubernetes/kubernetes/blob/91a9ce28ac2486c50222aeeec1f76e664155d769/test/e2e/storage/pvc_storageclass.go#L62
```

Comment:
Please link testgrid, there were few green runs in https://testgrid.k8s.io/sig-network-gce#gci-gce-alpha-features.

Reply:
Added.